store-implementation/proxy-store/src/test/resources/mock-proxy-store.properties

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright 2016 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+gaffer.store.class=uk.gov.gchq.gaffer.proxystore.SingleUseMockAccumuloProxyStore
+gaffer.store.properties.class=uk.gov.gchq.gaffer.proxystore.ProxyProperties
+gaffer.host=localhost
+gaffer.port=8080
+gaffer.context-root=/rest/v1
```

Comment:
This may need to change depending on the decision on underscores vs dashes from earlier.